pkg/kclient/operators.go

```diff
@@ -0,0 +1,16 @@
+package kclient
+
+import (
+	olmv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
```

Comment:
are we intended to support multiple versions here?

Reply:
No.

Reply:
i mean multiple versions at the same time?if not then why you have used `olmv1alpha1`?? `olm` would also suffice?

Reply:
That's because we're using the `v1alpha` version of OLM APIs. It kind of gives an idea that we're using a particular version of something. When we move to using `v1`, we can/should change it to `olmv1`.

Reply:
which version we are using could easily be inferred by the import statement `github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1`, when you will move to `v1` you need to change at every place you are using it.